[Qibo 0.2.12|INFO|2024-09-23 12:19:45]: Loading platform dummy
2024-09-23 12:19:46,367 - qm - INFO     - Starting session: cf49765c-9023-4d2d-b582-98e1dac1c7af
Traceback (most recent call last):
  File "/nfs/users/elisa.stabilini/test_venv/bin/qq", line 8, in <module>
    sys.exit(command())
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibocal/cli/_base.py", line 75, in auto
    autocalibrate(runcard, folder, force, update)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibocal/cli/autocalibration.py", line 20, in autocalibrate
    backend = construct_backend(backend=runcard.backend, platform=runcard.platform)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibo/backends/__init__.py", line 233, in construct_backend
    raise e
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibo/backends/__init__.py", line 229, in construct_backend
    return getattr(module, "MetaBackend").load(**kwargs)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibolab/backends.py", line 186, in load
    return QibolabBackend(platform=platform)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibolab/backends.py", line 45, in __init__
    self.platform = create_platform(platform)
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibolab/platform/load.py", line 64, in create_platform
    return _load(_search(name, _platforms_paths()))
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibolab/platform/load.py", line 44, in _load
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/nfs/tools/qibolab_platforms_qrc/qw11q/platform.py", line 4, in <module>
    from qibolab.instruments.qm import Octave, OPXplus, QMController
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibolab/instruments/qm/__init__.py", line 1, in <module>
    from .controller import QMController
  File "/nfs/users/elisa.stabilini/test_venv/lib/python3.10/site-packages/qibolab/instruments/qm/controller.py", line 11, in <module>
    from qualang_tools.simulator_tools import create_simulator_controller_connections
ModuleNotFoundError: No module named 'qualang_tools'
